homeassistant/components/aladdin_connect/cover.py

```diff
@@ -87,7 +97,7 @@ def __init__(
 
         self._device_id = device["device_id"]
         self._number = device["door_number"]
-        self._attr_name = device["name"]
+        self._name = device["name"]
```

Comment:
We also need to set `self._attr_has_entity_name` to True.

https://developers.home-assistant.io/docs/core/entity/#entity-naming

Reply:
That explains why the entity names were not "right"/changed - thanks for the feedback.